### PR TITLE
Fix typo in notes

### PR DESCRIPTION
**Pull request by 745150400**

### CODE REVIEW

File: lxd/endpoints/cluster.go [hunk 1]
```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/lxc/lxd/shared/logger"
 )
 
-// ClusterAddress returns the cluster addresss of the cluster endpoint, or an
+// ClusterAddress returns the cluster addresses of the cluster endpoint, or an
```
Comment (tomponline) [inline lxd/endpoints/cluster.go:14]: This should be just "address"

File: lxd/api.go [hunk 2]
```diff
@@ -17,7 +17,7 @@ import (
 
 // swagger:operation GET / server api_get
 //
-// Get the supported API enpoints
+// Get the supported API endpoints
```
Comment (tomponline) [inline lxd/api.go:20]: As this is used in the API docs we'll also need a separate commit after regenerating the API docs using `make update-api`, e.g. https://github.com/lxc/lxd/commit/77221976f405246b318af386baa244d4d87f1737

**Review by tomponline — CHANGES_REQUESTED** [review]
Thanks! A couple of small requests, then good to go :)

**Comment by stgraber** [comment]
jenkins: test this please

**Comment by stgraber** [comment]
jenkins: test this please

**Comment by stgraber** [comment]
Thanks!

**Pull request merged**